src/test/pybind/test_rados.py

```diff
@@ -312,8 +312,12 @@ def test_cmpext(self):
     def test_list_objects_empty(self):
         eq(list(self.ioctx.list_objects()), [])
 
-    def test_list_objects(self):
+    def test_read_crc(self):
         self.ioctx.write('a', b'')
+        self.ioctx.write('a', b'', 5)
+        self.ioctx.read('a')
+
+    def test_list_objects(self):
```

Comment:
Yeah, let's backport the commit from https://github.com/ceph/ceph/pull/57616 altogether.